index.html

```diff
@@ -98,96 +99,116 @@ <h2>The Finance and Investment club of NIT Trichy.</h2>
+          </p>
         </div>
 
         <div class="row content">
           <div class="col-lg-6">
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-              magna aliqua.
-            </p>
+            <p>The club conducts several events throughout the year to engage with the student community such as</p>
             <ul>
-              <li><i class="ri-check-double-line"></i> Ullamco laboris nisi ut aliquip ex ea commodo consequat</li>
-              <li><i class="ri-check-double-line"></i> Duis aute irure dolor in reprehenderit in voluptate velit</li>
-              <li><i class="ri-check-double-line"></i> Ullamco laboris nisi ut aliquip ex ea commodo consequat</li>
+              <li><i class="ri-check-double-line"></i>Knowledge Sharing Sessions</li>
+              <li><i class="ri-check-double-line"></i>Guest Lectures and Webinars</li>
+              <li><i class="ri-check-double-line"></i>Workshops</li>
+              <li><i class="ri-check-double-line"></i>Case Solving Competitions</li>
             </ul>
           </div>
           <div class="col-lg-6 pt-4 pt-lg-0">
-            <p>
-              Ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-              velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-              culpa qui officia deserunt mollit anim id est laborum.
-            </p>
-            <a href="#" class="btn-learn-more">Learn More</a>
-          </div>
+              <p>
+               The club also aims to empower its members by increasing their knowledge base in finance related domains, via various interactions,
+                discussions and industry projects brought with the help of our strong alumni network.
+              </p>
+
+              <div class="row-ig">
+                     <div class="column-ig">
+                       <a href = "https://icons8.com/icon/cc_e5716v_NK/bitcoin-cryptocurrency"><img class = "photogrid" src="https://img.icons8.com/ios/50/000000/bitcoin-cryptocurrency.png"></a>
+                     </div>
+                     <div class="column-ig">
+                       <a href = "https://icons8.com/icon/al0Ey554qDWT/company-assets"><img class = "photogrid" src="https://img.icons8.com/ios/50/000000/company-assets-.png"/></a>
+                     </div>
+                     <div class="column-ig">
+                       <a href = "https://icons8.com/icon/123468/dividends"><img class = "photogrid" src="https://img.icons8.com/ios/50/000000/dividends.png"/></a>
+                     </div>
+                     <div class="column-ig">
+                       <a href = "https://icons8.com/icon/63371/bonds"><img class ="photogrid" src="https://img.icons8.com/ios/50/000000/bonds.png"/></a>
+                     </div>
+               </div>
+            </div>

```

Comment:
1. Remove the random links
2. Use an icon font instead of images (check out what font they've used for other icons in this template)

Reply:
@9401adarsh were you not able to find icon fonts?

Reply:
@shyam-king I used icons from the same library the other icons were used hence the changes